dateparser_data/language_maps/update_language_maps.py

```diff
@@ -0,0 +1,32 @@
+import os
+import glob
+from pathlib import Path
+import json
+
+
+def main():
+    dateparser_data_dir = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+    all_files_name = glob.glob(dateparser_data_dir + "/cldr_language_data/date_translation_data/*.json")
+
+    all_language_code = [Path(file_name).stem for file_name in all_files_name]
+    all_language_code.sort()
+
+    data = {}
+
+    while all_language_code:
+        micro_data = []
+        for obj in all_language_code:
+            if not micro_data:
+                micro_data.append(obj)
+            else:
+                if micro_data[0] == obj[:len(micro_data[0])] and "-" in obj:
```

Comment:
I think ``micro_data[0] == obj[:len(micro_data[0])]`` can be simplified to something like ``obj.startswith(micro_data[0])``. Also don't we want to be more strict here and check for ``obj.startswith(micro_data[0] + '-')``?